features/step_definitions/catalogue_steps.rb

```diff
@@ -41,3 +41,9 @@
 Then(/^I see #{MAYBE_VAR} in the search summary text$/) do |value|
   find(:xpath, "//*[@class='search-summary']/em[1]").text().should == normalize_whitespace(value)
 end
+
+When /^I tick a random filter$/ do
+  checkbox = all(:xpath, "//form[@action='/g-cloud/search']//input[@type='checkbox']").sample
+  page.check(checkbox[:id])
```

Comment:
I think this still stands, but it'll be easier to implement after my new pull request (#269) goes in.